File: Raven.Client.Java/src/main/java/net/ravendb/client/document/RavenLoadConfiguration.java
package net.ravendb.client.document;

import java.util.HashMap;
import java.util.Map;

import net.ravendb.abstractions.json.linq.RavenJToken;
import net.ravendb.client.ILoadConfiguration;


public class RavenLoadConfiguration implements ILoadConfiguration {
  private Map<String, RavenJToken> queryInputs = new HashMap<>();

  public Map<String, RavenJToken> getQueryInputs() {
    return queryInputs;
  }

  public void setQueryInputs(Map<String, RavenJToken> queryInputs) {
    this.queryInputs = queryInputs;
  }

  @Override
  public void addQueryParam(String name, RavenJToken value) {
    queryInputs.put(name, value);
  }



}

File: src/Raven.NewClient/Document/RavenLoadConfiguration.cs
using System.Collections.Generic;
using Raven.NewClient.Json.Linq;

namespace Raven.NewClient.Client.Document
{
    public class RavenLoadConfiguration : ILoadConfiguration
    {
        public Dictionary<string, RavenJToken> TransformerParameters { get; set; }

        public RavenLoadConfiguration()
        {
            TransformerParameters = new Dictionary<string, RavenJToken>();
        }

        public void AddQueryParam(string name, RavenJToken value)
        {
            AddTransformerParameter(name, value);
        }

        public void AddTransformerParameter(string name, RavenJToken value)
        {
            TransformerParameters[name] = value;
        }
    }
}
